package.json

```diff
@@ -43,37 +43,37 @@
   },
   "devDependencies": {
     "@cdk8s/projen-common": "0.0.530",
-    "@cdktf/provider-aws": "^17.0.11",
```

Comment:
Hmm, is this required by `cdktf@0.20.7`?

Reply:
Yes. The version constraint on the provider package is incompatible with 0.20.7 of the main cdktf package.

Reply:
CDKTF version 0.20.0 support was added to @cdktf/provider-aws starting @ v19.0.0 which released January 10, 2024. [Commit](https://github.com/cdktf/cdktf-provider-aws/commit/e717f618b5852e198ca240f84b1043cb6b9956d0) [Release](https://github.com/cdktf/cdktf-provider-aws/releases/tag/v19.0.0)

The package.json for the release included [typescript ~5.2.0](https://github.com/cdktf/cdktf-provider-aws/blob/e717f618b5852e198ca240f84b1043cb6b9956d0/package.json#L62C6-L62C16).